apps/vault/contracts/Vault.sol

```diff
@@ -94,7 +97,7 @@ contract Vault is EtherTokenConstant, AragonApp, DepositableStorage {
             // Deposit is implicit in this case
             require(msg.value == _value, ERROR_VALUE_MISMATCH);
         } else {
-            require(ERC20(_token).transferFrom(msg.sender, this, _value), ERROR_TOKEN_TRANSFER_FROM_REVERTED);
+            require(ERC20(_token).safeTransferFrom(msg.sender, this, _value), ERROR_TOKEN_TRANSFER_FROM_REVERTED);
```

Comment:
Nit, `this` references to the contract instance, it's more appropiate to do `address(this)` in  this case. Same applies for the rest of the incidences referencing `this` as the address of the contract.

Reply:
Will fix in another PR.